@xen-orchestra/fs/src/abstract.js

```diff
@@ -31,6 +32,9 @@ const computeRate = (hrtime: number[], size: number) => {
 }
 
 const DEFAULT_TIMEOUT = 6e5 // 10 min
+const DEFAULT_MAXIMUM_PARALLEL_OPERATIONS = 10
+
+let maximum_operations = DEFAULT_MAXIMUM_PARALLEL_OPERATIONS
```

Comment:
Since when do we use global variables?

Reply:
because this doesn't work => 
```js
@limit(this._maximumParallelOperations)
```

Reply:
Yes it does not, you need to the *function wrapper* syntax.